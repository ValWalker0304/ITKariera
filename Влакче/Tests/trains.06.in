Add 37643 Karlovo P 7
Add 61122 Ruse F 12
Add 99389 Montana P 1
Next
Next
Travel
Add 12501 StaraZagora P 9
Add 95806 Targovishte F 36
Travel
Travel
Travel
Travel
Next
Next
Next
Add 57143 Burgas F 18
Travel
Next
Travel
Add 49648 Ruse P 5
Next
Add 88587 Ruse F 40
Next
Add 84149 Burgas P 5
Travel
Travel
Next
Add 77428 Sindel F 13
Add 78035 Kulata F 11
Add 15031 Svilengrad P 6
Travel
Travel
Next
End