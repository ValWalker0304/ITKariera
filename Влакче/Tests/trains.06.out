99389 Montana P 1
99389 Montana P 1
99389 Montana P 1
95806 Targovishte F 36
12501 StaraZagora P 9
37643 Karlovo P 7
61122 Ruse F 12
57143 Burgas F 18
49648 Ruse P 5
88587 Ruse F 40
88587 Ruse F 40
84149 Burgas P 5
49648 Ruse P 5
15031 Svilengrad P 6
49648 Ruse P 5
78035 Kulata F 11
